apps/pages/vite.config.js

```diff
@@ -1,16 +1,10 @@
-import { defineConfig } from "vite";
-import { createVuePlugin as vue } from "vite-plugin-vue2";
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue'
 
-// https://vitejs.dev/config/
 export default defineConfig({
   plugins: [vue()],
-  resolve: {
-    alias: {
-      vue: require.resolve("vue/dist/vue.js"),
-    },
-  },
   base: "",
   server: {
     proxy: require("../dev-proxy.config"),
-  },
+  }
```

Comment:
The layouting of your .js files is different from the .vue files

Reply:
yes, I get it, we must apply an external automformatter so we don't have issues between our different editors. I made mistake of opening these files in other IDE instance then normal.

Reply:
Rerun prettier on changed files